perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -65,6 +65,11 @@ sub index : Path : Args(0) {
         unless ($children->{error}) {
             $c->stash->{children} = $children;
         }
+        my $covered = {};
+        $c->cobrand->call_hook('add_parish_wards' => $covered);
+        unless ($covered->{error}) {
+            $c->stash->{covered} = $covered;
+        }
```

Comment:
I think calling this variable and the stash item (and by extension the template variable) "covered" is a bit confusing. It made sense for Oxfordshire, because the data was coming from a Mapit covers call, but other councils might get the data in a different way.

Perhaps something like `parish_wards` or similar would be a bit clearer?